packages/core/parcel/src/bin.js

```diff
@@ -1,2 +1,9 @@
-#!/usr/bin/env node -r @parcel/babel-register
+#!/usr/bin/env node
+
+'use strict';
```

Comment:
We configure Babel to compile everything as if it were an ES module, so this is set for consistency between the source and the compiled file.